README.md

```diff
@@ -133,7 +133,7 @@ Taking a container object and turning it into a runnable process on a system is
 
 ```go
 // create a new task
-task, err := redis.NewTask(context, containerd.Stdio)
+task, err := redis.NewTask(context, cio.Stdio)
```

Comment:
When `containerd.Client` is introduced further up this file a `import "github.com/containerd/containerd"` is included to give the context. Do we want/need a similar `import "github.com/containerd/containerd/cio"` to give context for the use of `cio.Stdio`?